doc/decoding/going_further.rst

```diff
@@ -0,0 +1,156 @@
+.. _going_further:
+
+==========================================================================
+Running scikit-learn functions for more control on the analysis
+==========================================================================
+
+This section gives pointers to design your own decoding pipelines with
+scikit-learn. This builds on the :ref:`didactic introduction to decoding <decoding_intro>`.
+
+.. note::
+  This documentation gives links and additional definitions needed to work
+  correctly with scikit-learn. For a full code example, please check out: :ref:`sphx_glr_auto_examples_07_advanced_plot_advanced_decoding_scikit.py`
+
+
+.. contents:: **Contents**
+    :local:
+    :depth: 1
+
+
+Performing decoding with scikit-learn
+=======================================
+
+Using scikit-learn estimators
+--------------------------------
+
+You can easily import estimators from the `scikit-learn <http://scikit-learn.org>`
+machine-learning library, those available in the `Decoder` object and many
+others. They all have the `fit` and `predict` functions. For example you can
+directly import the versatile `Support Vector Classifier <http://scikit-learn.org/stable/modules/svm.html>`_ (or SVC).
+
+To learn more about the variety of classifiers available in scikit-learn,
+see the `scikit-learn documentation on supervised learning
+<http://scikit-learn.org/stable/supervised_learning.html>`_).
+
+
+Cross-validation with scikit-learn
+-----------------------------------
+
+To perform cross-validation using a scikit-learn estimator, you should first
+mask the data using a :class:`nilearn.input_data.NiftiMasker`: to extract
+only the voxels inside the mask of interest, and transform 4D input fMRI
+data to 2D arrays (shape (n_timepoints, n_voxels)) that estimators can work on.
```

Comment:
Maybe point to one of our examples here (in a topic box, to make it more visible)